[2024-03-29 13:10:46,843] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:10:46,846] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:10:46,849] INFO _log() L96    * Restarting with stat - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:10:49,222] WARNING _log() L96    * Debugger is active! - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:10:49,231] INFO _log() L96    * Debugger PIN: 142-274-112 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:02,971] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:02] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:03,022] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:03] "GET /openapi/ HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:03,218] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:03] "GET /openapi/static/images/redoc.svg HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:03,223] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:03] "GET /openapi/static/images/swagger.svg HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:03,228] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:03] "GET /openapi/static/images/rapidoc.svg HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:03,238] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:03] "GET /openapi/static/images/apidoc.svg HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:04,745] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:04] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:04,787] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:04] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:04,793] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:04] "GET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:04,807] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:04] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:05,051] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:05] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:05,059] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:05] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:12,111] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:12] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-03-29 13:11:23,745] INFO _log() L96   127.0.0.1 - - [29/Mar/2024 13:11:23] "[35m[1mGET /produto?nome=Teste HTTP/1.1[0m" 500 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:12,881] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:12,884] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:12,889] INFO _log() L96    * Restarting with stat - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:15,386] WARNING _log() L96    * Debugger is active! - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:15,391] INFO _log() L96    * Debugger PIN: 142-274-112 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 09:59:20,064] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 09:59:20] "[35m[1mGET /produto?nome=Teste HTTP/1.1[0m" 500 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:01:58,823] INFO _log() L96    * Detected change in 'c:\\Users\\ricar\\OneDrive\\Cursos\\Pyton\\desenvolvimento-full-stack\\desenvolvimento-full-stack-basico\\aula-3\\meu_app_api\\app.py', reloading - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:01,786] INFO _log() L96    * Restarting with stat - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:16,110] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:16,112] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:16,116] INFO _log() L96    * Restarting with stat - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:18,488] WARNING _log() L96    * Debugger is active! - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:18,493] INFO _log() L96    * Debugger PIN: 142-274-112 - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:02:54,843] WARNING get_produto() L104  Erro ao buscar produto 'Teste', Produto não encontrado na base :/ - call_trace=c:\Users\ricar\OneDrive\Cursos\Pyton\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py L104 
[2024-04-01 10:02:56,717] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 10:02:56] "[33mGET /produto?nome=Teste HTTP/1.1[0m" 404 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:03:48,709] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 10:03:48] "GET /openapi/static/images/swagger.svg HTTP/1.1" 200 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:03:51,205] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 10:03:51] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:04:04,251] WARNING del_produto() L135  Erro ao deletar produto #'Teste', Produto não encontrado na base :/ - call_trace=c:\Users\ricar\OneDrive\Cursos\Pyton\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py L135 
[2024-04-01 10:04:04,255] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 10:04:04] "[33mDELETE /produto?nome=Teste HTTP/1.1[0m" 404 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:04:26,475] WARNING del_produto() L135  Erro ao deletar produto #'Jiló', Produto não encontrado na base :/ - call_trace=c:\Users\ricar\OneDrive\Cursos\Pyton\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py L135 
[2024-04-01 10:04:26,477] INFO _log() L96   127.0.0.1 - - [01/Apr/2024 10:04:26] "[33mDELETE /produto?nome=Jiló HTTP/1.1[0m" 404 - - call_trace=c:\Users\ricar\AppData\Local\Programs\Python\Python312\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-01 10:05:25,894] WARNING del_produto() L135  Erro ao deletar produto #'Jiló', Produto não encontrado na base :/ - call_trace=c:\Users\ricar\OneDrive\Cursos\Pyton\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py L135 
